src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -168,61 +172,88 @@ export class ConnectionWidget extends lifecycle.Disposable {
 		if (this._authTypeSelectBox) {
 			this.onAuthTypeSelected(this._authTypeSelectBox.value);
 		}
-
-		DOM.addDisposableListener(container, 'paste', e => {
-			this._handleClipboard().catch(err => this._logService.error(`Unexpected error parsing clipboard contents for connection widget : ${err}`));
```

Comment:
If we're removing this we should remember to call that out in the release docs in case people are actually using this functionality so they aren't caught off guard by it stopping working suddenly.

Reply:
I'll drop PM a note about this feature.